src/app/ubs-user/ubs-user-orders-list/ubs-user-orders-list.component.ts

```diff
@@ -0,0 +1,19 @@
+import { Component, Input } from '@angular/core';
+
+@Component({
+  selector: 'app-ubs-user-orders-list',
+  templateUrl: './ubs-user-orders-list.component.html',
+  styleUrls: ['./ubs-user-orders-list.component.scss']
+})
+export class UbsUserOrdersListComponent {
+  @Input()
+  orders: any[];
+
+  constructor() {}
+
+  changeCard(id: number) {
+    this.orders.map((el) => {
+      el.id === id && el.extend ? (el.extend = false) : el.id === id ? (el.extend = true) : (el.extend = el.extend);
```

Comment:
It's better to use understandable variable
Suggestions:
'order'

Reply:
done